cukinia

```diff
@@ -116,6 +127,12 @@ cukinia_http_request()
 	cukinia_runner _cukinia_http_request "$1"
 }
 
+# not: negate the issue of what follows
+not()
+{
+	__not=1 "$@"
```

Comment:
* ! and 'set -e' does not work. We don't want cukinia to terminate when a test pass. We want it to be ++'ed and the other tests to continue
*  'local' is not specified by posix, however it is accepted by dash and BSD 'sh', which are our lowest common denominator in terms of language support, so no change at the header level unless you can foresee a scenario where the execution breaks.